apps/docs/content/components/forms/ErrorMessage.mdx

```diff
@@ -1,48 +1,62 @@
 ---
-title: ErrorMessage (WIP)
-description: TBD
+title: ErrorMessage
+description: An ErrorMessage displays validation errors for a form field.
 category: "forms"
 links:
-    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/errorMessage/src/ErrorMessage.tsx
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/ErrorMessage/src/ErrorMessage.tsx
 ---
 
+<Example src="ErrorMessage/docs/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+<PropTable component="ErrorMessage" />
 
-## Guidelines
+<FeatureFlag flag="next">
+    ## Guidelines
 
-TODO: If we have some guidelines about this component's usage
+    TODO: If we have some guidelines about this components usage
```

Comment:
```suggestion
    TODO: If we have some guidelines about this component's usage
```
It should have the apostrophe. I read it a couple times too and was like hmm..
I have it like this in mine for some reason: component&apos;s